src/System.Data.Common/tests/System/Data/Common/DbExceptionTests.cs

```diff
@@ -8,22 +8,70 @@ namespace System.Data.Common.Tests
 {
     public class DbExceptionTests
     {
+        [Fact]
+        public void DefaultArgumentsShowed()
+        {
+            var defaultException = new CustomDbException();
+            Assert.Equal("External component has thrown an exception.", defaultException.Message);
```

Comment:
We generally don't validate error message likes this, as they may get localized, in which case the test will fail when run on non-English cultures.  You can instead just do a NonNull test, or assert that string.IsNullOrWhitespace is false, or something like that.